### PR TITLE
Update new-scheduledtasksettingsset.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#548 
Step Taken: 
- Added #6 example to -ExecutionTimeLimit 
-ExecutionTimeLimit (New-TimeSpan -Hours 1) same as ExecutionTimeLimit =  'PT1H'
it specify that if the task is not finished after 1 hour, it is considered as failed, it need to finish the task after 1 hour.

-Also added that "Without the ExecutionTimeLimit setting defined the time limit set to it's default of 3 days for the Task Scheduler is allowed to complete the task."
-added article for time limit parameter https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-timespan?view=powershell-6

References;
https://social.technet.microsoft.com/Forums/en-US/26be656a-ce4f-4566-a994-0e0c2edabac8/windows-task-scheduler-time-limit-set-to-unlimited?forum=winserver8gen

The -ExecutionTimeLimit parameter is for New-ScheduledTaskSettingsSet command.
https://social.technet.microsoft.com/Forums/en-US/26be656a-ce4f-4566-a994-0e0c2edabac8/windows-task-scheduler-time-limit-set-to-unlimited?forum=winserver8gen

https://social.technet.microsoft.com/Forums/en-US/26be656a-ce4f-4566-a994-0e0c2edabac8/windows-task-scheduler-time-limit-set-to-unlimited?forum=winserver8gen

Trigger task scheduler Job from powershell
https://stackoverflow.com/questions/44491658/trigger-task-scheduler-job-from-powershell?rq=1

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/scheduledtasks/new-scheduledtasksettingsset.md [hunk 1]
```diff
@@ -121,6 +121,19 @@ This example registers a scheduled task that runs only when a network is availab
...
+```
+PS C:\>$Sta = New-ScheduledTaskAction -Execute "Cmd"
+
+The second command creates scheduled task settings that specify if the task is not finished after 1 hour, it is considered as failed. This command assigns the **ScheduledTaskSettings** object to the $Stset variable.
+$Stset = New-ScheduledTaskSettingsSet -ExecutionTimeLimit (New-TimeSpan -Hours 1)
+
+The third command registers the scheduled task Task01 to run the task action named Cmd only then finish the task after 1 hour.
+PS C:\>Register-ScheduledTask Task01 -Action $Sta -Settings $Stset
+```
+Without the ExecutionTimeLimit setting defined the time limit set to it's default of 3 days for the Task Scheduler is allowed to complete the task. To configure the time limit, see [New-TimeSpan](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-timespan?view=powershell-6).
```
Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
Without the ExecutionTimeLimit setting defined the time limit set to its default of 3 days for the Task Scheduler is allowed to complete the task. To configure the time limit, see [New-TimeSpan](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-timespan?view=powershell-6).
```

Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
Without the ExecutionTimeLimit setting defined the time limit set to it's default of three days for the Task Scheduler is allowed to complete the task. To configure the time limit, see [New-TimeSpan](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-timespan?view=powershell-6).
```

Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
Without the ExecutionTimeLimit setting defined, the time limit set to its default of three days for the Task Scheduler is allowed to complete the task. To configure the time limit, see [New-TimeSpan](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-timespan?view=powershell-6).
```

File: docset/windows/scheduledtasks/new-scheduledtasksettingsset.md [hunk 2]
```diff
@@ -121,6 +121,19 @@ This example registers a scheduled task that runs only when a network is availab
 
 The first command creates a scheduled task action named Cmd and assigns the **ScheduledTaskAction** object to the $Sta variable.
 
+### Example 6: Register a scheduled task that has a time limit to complete the task
+```
+PS C:\>$Sta = New-ScheduledTaskAction -Execute "Cmd"
+
+The second command creates scheduled task settings that specify if the task is not finished after 1 hour, it is considered as failed. This command assigns the **ScheduledTaskSettings** object to the $Stset variable.
```
Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
The second command creates scheduled task settings that specify if the task is not finished after one hour, it is considered as failed. This command assigns the **ScheduledTaskSettings** object to the $Stset variable.
```

File: docset/windows/scheduledtasks/new-scheduledtasksettingsset.md [hunk 3]
```diff
@@ -121,6 +121,19 @@ This example registers a scheduled task that runs only when a network is availab
...
+$Stset = New-ScheduledTaskSettingsSet -ExecutionTimeLimit (New-TimeSpan -Hours 1)
+
+The third command registers the scheduled task Task01 to run the task action named Cmd only then finish the task after 1 hour.
```
Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
The third command registers the scheduled task Task01 to run the task action named Cmd only then finish the task after one hour.
```

Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/new-scheduledtasksettingsset.md]:
```suggestion
The third command registers the scheduled task Task01 to run the task action named Cmd only, then finish the task after one hour.
```

**Comment by e0i** [comment]
@sukihanabi Please incorporate suggested copy changes into your PR. Thanks.

**Pull request merged**